roles/container_runtime/tasks/package_docker.yml

```diff
@@ -31,6 +31,20 @@
   register: result
   until: result is succeeded
 
+ - name: Install cri-tools
+   package:
+    name: "cri-tools"
+    state: present
```

Comment:
Any reason not to say latest? This ships in the per release repos, right?

Reply:
Sure, I'll fix that

Reply:
Fixed.